product_supplierinfo_for_customer/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+    :target: http://www.gnu.org/licenses/agpl-3.0.en.html
+
+============================================
+Use product supplier info also for customers
+============================================
+
+This modules allows to use supplier info structure, available in
+*Inventory* tab of the product form, also for defining customer information,
+allowing to define prices per customer and product.
+
+Configuration
+=============
+
+For these prices to be used in sale prices calculations, you will have
+to create a pricelist with a rule with option "Based on" with the value
+"Supplier prices on the product form" (although the text is not clear enough).
+
+Usage
+=====
+
+There's a new section on *Sales* tab of the product form called "Customers",
+where you can define records for customers with the same structure of the
+suppliers.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/188/9.0
+
+
+The product code / product name specified for the customer can be reflected
+on the sale orders using module `product_supplierinfo_for_customer_sale
+<https://github.com/OCA/product-attribute/tree/8.0/product_supplierinfo_for_customer_sale>`_
```

Comment:
s/8.0/9.0 You can use the new README by fragments system for avoiding to change this.

Reply:
that's not my PR. I will do fragments maybe in new modules for v11.

Reply:
Then the version should be corrected. You can apply the system the same for <v11. It's more comfortable IMO, but as you prefer.

Reply:
the version has been corrected.

And I am not comfortable with the fragments yet 🙄